sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/networking.py

```diff
@@ -184,13 +185,31 @@ class ManagedNetworkSchema(metaclass=PatchedSchemaMeta):
         ),
         allow_none=True,
     )
+    firewall_sku = StringTransformedEnum(
+        allowed_values=[
+            FirewallSku.STANDARD,
+            FirewallSku.BASIC,
+        ],
+        casing_transform=camel_to_snake,
+        metadata={"description": "Firewall sku for FQDN rules in AllowOnlyApprovedOutbound mode"},
+    )
     network_id = fields.Str(required=False, dump_only=True)
     status = NestedField(ManagedNetworkStatusSchema, allow_none=False, unknown=EXCLUDE)
 
     @post_load
     def make(self, data, **kwargs):
         outbound_rules = data.get("outbound_rules", False)
+
+        firewall_sku = data.get("firewall_sku", False)
+        firewall_sku_value = _snake_to_camel(data["firewall_sku"]) if firewall_sku else None
```

Comment:
instead of default to None when unspecified, should we be setting a default value here or we prefer to set in service side?

Reply:
changed to standard